cmd/thv/app/run_config.go

```diff
@@ -0,0 +1,292 @@
+package app
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	cfg "github.com/stacklok/toolhive/pkg/config"
+	"github.com/stacklok/toolhive/pkg/container"
+	"github.com/stacklok/toolhive/pkg/container/runtime"
+	"github.com/stacklok/toolhive/pkg/process"
+	"github.com/stacklok/toolhive/pkg/registry"
+	"github.com/stacklok/toolhive/pkg/runner"
+	"github.com/stacklok/toolhive/pkg/runner/retriever"
+	"github.com/stacklok/toolhive/pkg/transport"
+	"github.com/stacklok/toolhive/pkg/transport/types"
+)
+
+// RunConfig holds the configuration for running MCP servers
+type RunConfig struct {
```

Comment:
Since we already have a `RunConfig` struct elsewhere in the codebase, can this one be `RunParams` or `RunFlags`?